docs/changelog.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 2
-sidebar_label: Full Changelog
+sidebar_label: Helm 4 Full Changelog
```

Comment:
so that the sidebar label matches the page title (not that this is required, but now that all the existing docs content is ported over, it seemed beneficial to be able to see "Helm 4" in the title in the sidebar to better understand what you're clicking on)

Reply:
I missed that you already did this one. from https://github.com/helm/helm-www/pull/1875#pullrequestreview-3419004874:

> Previously, this content was in the top level `docs/index.mdx` file, which is called Docs Home in the Helm 2 & 3 docs. Moved this content to "Helm 4 Overview" instead to consolidate into one page.

disregard that part of my review 🙂